pkg/cmd/server/kubernetes/node.go

```diff
@@ -421,14 +423,29 @@ func (c *NodeConfig) RunProxy() {
 	default:
 		glog.Fatalf("Unknown proxy mode %q", c.ProxyConfig.Mode)
 	}
-	iptInterface.AddReloadFunc(proxier.Sync)
 
 	// Create configs (i.e. Watches for Services and Endpoints)
 	// Note: RegisterHandler() calls need to happen before creation of Sources because sources
 	// only notify on changes, and the initial update (on process start) may be lost if no handlers
 	// are registered yet.
 	serviceConfig := pconfig.NewServiceConfig()
-	serviceConfig.RegisterHandler(proxier)
+
+	unidlingLoadBalancer := ouserspace.NewLoadBalancerRR()
```

Comment:
Is it acceptable to add additional proxy modes in an OpenShift specific place?  If so, I could just add a "idling-iptables" and "idling-userspace" proxy modes.  Otherwise, I'll just add a flag.